link/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -35,6 +36,8 @@ internal class WalletViewModel @Inject constructor(
             paymentDetailsList = emptyList(),
             selectedItem = null,
             isProcessing = false,
+            hasCompleted = false,
+            primaryButtonLabel = completePaymentButtonLabel(configuration.stripeIntent)
```

Comment:
since `completePaymentButtonLabel` is only used in this class, can it be a member of this class instead of of PrimaryButton?